index.html

```diff
@@ -1306,9 +1306,9 @@ <h3>
           </tbody>
         </table>
         <p>
-          The <a>media type for a manifest</a> serves as the default media type
-          for resources associated with the "<code>manifest</code>" <a>link
-          type</a>.
+          The <a>MIME type for a manifest</a> serves as the default
+          <a data-cite="mimesniff">MIME type</a> for resources associated with
```

Comment:
Nit: we can move `data-cite=` to the `<section>` (or even up to the `<body>`), then we don't need to pepper the spec with so many `data-cite=`s.

Reply:
Done. Moved it to the various sections (I didn't want to pollute the entire body).